_events/sr2017/southampton-tech-day-december.md

```diff
@@ -12,7 +12,14 @@ The Southampton tech days take advantage of the facilities of the University of
 
 ## Directions
 
-This years tech day will be at the University of Southampton, the exact location will be announced soon!
+The location of our tech day is on a different campus to that of Kickstart, please check our map!
```

Comment:
fixed in a018298 Initially it was only the first one since kickstart, but I saw my error in assuming all tams go to the tech days, and added it to all of them.